scripts/veloxbench/veloxbench/cpp_micro_benchmarks.py

```diff
@@ -15,6 +15,8 @@
 import copy
```

Comment:
like @jonkeane mentioned above, could we keep these two scripts separated so that we could still run the benchmark test suite locally without the conbench dependency?

Maybe the API could look something like:

> ./scripts/benchmark-runner.py -> runs all benchmarks locally, prints to stdout 
> ./scripts/conbench-runner.py -> runs all benchmarks and uploads results to conbench

where the first one doesn't depend on conbench. It could also make sense to refactor the common functions into a scripts/benchmark-common.py library of some sort.